docs/conf.py

```diff
@@ -22,21 +22,38 @@
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 
 MOCK_MODULES = [
+    "CairoSVG",
```

Comment:
```suggestion
```
This is a sphinx module and I don't think it should be here.

Reply:
This was removed